src/HPWH.hh

```diff
@@ -282,7 +282,10 @@ class HPWH : public Courier::Sender
 
         MODELS_AquaThermAire = 400, // heat exchanger model
 
-        MODELS_GenericUEF217 = 410
+        MODELS_GenericUEF217 = 410,
+        MODELS_AeroTherm2023_50 = 411,
+        MODELS_AeroTherm2023_65 = 412,
+        MODELS_AeroTherm2023_80 = 413
```

Comment:
Start a new series (500?) for Bradford-White AeroTherm. Follow same naming convention as other enumerations:

`MODELS_BradfordWhiteAeroThermRE2H50`

Reply:
Done.

Reply:
Done.